docs/contributing/index.md

```diff
@@ -1,5 +1,12 @@
 # Contributing
 
+## Contribution flow
+
+Did you find a software bug? -> Open an issue in the corresponding software repository
+Did you find strange behavior? -> [Ask in Q&A](https://github.com/autowarefoundation/autoware/discussions/categories/q-a)
+Do you have questions about Autoware functionality? -> [Ask in Q&A](https://github.com/autowarefoundation/autoware/discussions/categories/q-a)
+Do you want to add new features to Autoware? -> [Make a feature request](https://github.com/autowarefoundation/autoware/discussions/categories/feature-requests)
```

Comment:
@LalithVipulananthan Could you please give me your ideas about how to write these things on this page? :pray:

Reply:
@kenji-miyake Although I agree that all of the information about how to contribute and how to get support is there in the documentation, I can also see where Ishita-san's point that it may be a little confusing since there is overlap between contributing and support. 

To me, contributing is:
- Participating in the community (Discord) and discussions (GitHub discussions)
- Joining an AWF working group
- Reviewing PRs
- Assisting other people with problems (providing support, rather than asking for support)
(Reference: http://wiki.ros.org/Get%20Involved)

Support should cover:
- What to do if you have a problem
- What to do if you have a fix for an issue
- What to do if you have a technical question
- What to do if you have confirmed that an issue is a reproduceable bug
(Reference: http://wiki.ros.org/Support)

Making PRs and creating issues is also a form of contribution, but they follow on from "What do I do if I have a problem or a question?" and so I think that it would be better to include these things as part of the Support page.

I propose the following:
- The Contributing page should talk about Discord, GitHub Discussions and the AWF working groups
- The Support page should talk about what to do when you have a problem (most of what Ishita-san mentions above), and then explain the guidelines for asking a question, creating an issue or submitting a PR
  - We should rename the "Help" page to "Support" as this is the standard term used in most places
- The Code of Conduct should be split off into its own page at the top of the site (next to Support for example). 
  - Doing so makes it clear to contributors that we have a code of conduct and that we take it seriously.
  - Separating the code of conduct from the Contributing page allows us to reduce the length of the Contributing page

Assuming the above is acceptable, I am happy to submit a PR with these changes since I appreciate that it's not a small amount of work and goes well beyond the original request for feedback!

Reply:
It's going to be a large amount of work. Should we continue the structure modification in this PR?

Reply:
You can try it if you are interested.
Or Lalith-san will create another PR about his proposal?

Reply:
@IshitaTakeshi I was planning to submit a new PR with my proposed changes. It might take me a little while, but I was aiming to get a first cut ready for next week.

Reply:
@LalithVipulananthan OK. I'm going to continue this after the structure modification, or we can apply the modification in this PR to your PR since the file changes are still small